src/chains/models.py

```diff
@@ -33,6 +33,7 @@ class RpcAuthentication(models.TextChoices):
         default=100
     )  # A lower number will indicate more relevance
     name = models.CharField(verbose_name="Chain name", max_length=255)
+    short_name = models.CharField(verbose_name="EIP-3770 short name", max_length=255)
```

Comment:
This could even be unique (and maybe should to avoid issues in the future)

Reply:
I agree 👍 Good point. I will add it.